src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -387,6 +386,8 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
               }
             } map {
               case RequestComplete(StatusCodes.NoContent) =>
+                //this is a case where we don't want to rollback the transaction in the event of getting an error.
```

Comment:
Move this comment down a few lines to the ErrorReport, and add that this is why it is not an exception.